### PR TITLE
#1378 Fix openpgp types issue

### LINKED ISSUE #1378: fix types issue from OpenPGP.js v5 encrypt armored/binary
from #1374 

image

### PR DESCRIPTION
**Pull request by sosnovsky**
This PR simplifies code for `openpgp` encrypt

close #1378

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: Core/source/core/pgp-msg.ts [hunk 1]
```diff
@@ -298,27 +298,20 @@ export class PgpMsg {
...
+    let encryptOptions = {
+      message,
+      date,
+      encryptionKeys,
+      passwords: pwd ? [pwd] : undefined,
+      signingKeys: signingPrv && signingPrv.isPrivate() ? signingPrv : undefined
+    };
+
+    if (!armor) {
+      encryptOptions = Object.assign(encryptOptions, { format: 'binary' });
```
Comment (ioanmo226) [inline Core/source/core/pgp-msg.ts:311]: Better to use something like this.

```ts
encryptOptions = {
    ...encryptOptions,
    format: 'binary'
}

```

Comment (sosnovsky) [reply Core/source/core/pgp-msg.ts:311]: @ioanmo226 I tried to use it, but working code looks like this (as for me `Object.assign` syntax looks simpler in this case)

```ts
encryptOptions = {
    ...encryptOptions,
    ...{ format: 'binary' }
};
```

Comment (ioanmo226) [reply Core/source/core/pgp-msg.ts:311]: Ok, got it

**Review by ioanmo226 — APPROVED** [review]
👍

**Pull request merged**